### ISSUE TITLE
Mage - Blast Wave knock up

### ISSUE THREAD

**Issue by kysqt** [dalaranwow/dalaran-wow#5230]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: So I've noticed blast wave is a little off from blizzlike, the knock up range is not what it should be which puts fire mages at a disadvantage in pve and pvp. I tested in 2v2 skrim to confirm something with my mate, Blast wave knock up has the ability in ruins of lordaeron to knock a player from the side of the tomb to the top of it, allowing the mage to make the current target LOS a healer https://gyazo.com/53539da79394cd2409072746f3e61cec <<this for reference. Was a common thing for skilled mages to do, I cannot find anything online about it but remember several mages doing it to me back in the day when fire mages and rogues were a thing.

This simply isn't possible on dalaran-wow as we tested, looking online I cannot find anything saying the exact knock up yards of Blast wave but there are youtube videos of it, you can clearly see the difference https://www.youtube.com/watch?v=xUv4sfZFVhc https://www.youtube.com/watch?v=NbGSziBc16c It's a bit difficult to find a good example in pvp videos because most of them are not on a flat surface. But if you compare videos of blast wave to dalaran-wow gameplay you will notice a difference.

I just tested it on warmanes instant 80 server and indeed it is possible over there, not that warmane is a solid reference point but I basically cannot compare it to anything else.

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. 
2.
3. 

**Include proofs for this behaviour**
https://www.youtube.com/watch?v=xUv4sfZFVhc
https://www.youtube.com/watch?v=NbGSziBc16c

**You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
